### PR TITLE
fix(typescript): remove duplicate capture

### PR DESCRIPTION
**Pull request by amaanq**
It's in ecma already (added a missing operator too now)

### CODE REVIEW

File: queries/typescript/highlights.scm [hunk 1]
```diff
@@ -38,6 +38,8 @@
 
 (template_literal_type) @string
 
+"!" @operator
```
Comment (lucario387) [inline queries/typescript/highlights.scm:41]: If anything, I'd say this should also be in ecma/highlights. Don't see why only typescript is allowed to have this highlighted

Comment (amaanq) [reply queries/typescript/highlights.scm:41]: well, in ecma it's distinguished on L257:
```(unary_expression ["!" "~" "-" "+"] @operator)```

Comment (lucario387) [reply queries/typescript/highlights.scm:41]: Yeah, but I don't remember any explicit case where "!" can be used in typescript that is not in javascript tho

Comment (amaanq) [reply queries/typescript/highlights.scm]:
```ts
const result = maybeundefinedreturn()!
```

Comment (lucario387) [reply queries/typescript/highlights.scm]: Well I don't know of that bizarre syntax :smile:

**Pull request merged**